src/modules/kubernetes.rs

```diff
@@ -955,4 +981,135 @@ users: []
         assert_eq!(expected, actual);
         dir.close()
     }
+
```

Comment:
https://github.com/starship/starship/pull/614 contains more tests, I tried to port the most relevant over. Not sure if I should port more?